convert2rhel/checks.py

```diff
@@ -147,7 +147,8 @@ def check_readonly_mounts():
 
 
 def check_custom_repos_are_valid():
-    """To prevent failures past the PONR, make sure that the enabled custom repositories are valid.
+    """To prevent failures past the PONR, make sure that the enabled custom
+    repositories are valid.
```

Comment:
Why this change? The line was below 120 chars. Ideally, to follow [PEP 257](https://www.python.org/dev/peps/pep-0257/), the multiline docstring of functions start with a single summary line, followed by an empty line and then detailed description.